### PR TITLE
ahmed2022re 

### PR DESCRIPTION
**Pull request by Divyasha02**
## Project Name

Write the `<ahmed2022re>` here, and as well as rename the PR name to reflect the `<ahmed2022re>` for quick reviewing. 

## Compiled PDF

Add the link to the compiled pdf here. Example : [Compiled Pdf](2021/<ahmed2022re>/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

```
Building ahmed2022re/
/home/lenovo/MLRC/2021/ahmed2022re/journal
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
rm: cannot remove 'metadata.tex': No such file or directory
/usr/bin/env: ‘python3\r’: No such file or directory
make: *** [Makefile:18: metadata.tex] Error 127
Build complete
rm: cannot remove 'article.aux': No such file or directory
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory
rm: cannot remove 'article.fdb_latexmk': No such file or directory
rm: cannot remove 'article.fls': No such file or directory
rm: cannot remove 'article.out': No such file or directory
rm: cannot remove 'article.log': No such file or directory
rm: cannot remove 'article.run.xml': No such file or directory
rm: cannot remove 'article.bcf': No such file or directory
Building editorial/
/home/lenovo/MLRC/2021/editorial/journal
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl'
 No file article.bbl.
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation 'Roboto:2011' on page 2 undefined on input line 74
  Citation 'Rougier:2017' on page 1 undefined on input line 12
  Citation 'Rougier:2017' on page 1 undefined on input line 26
  Citation 'Rougier:2018' on page 1 undefined on input line 13
  Citation 'Science:2018' on page 1 undefined on input line 11
  Citation 'SourceCodePro:2012' on page 2 undefined on input line 74
  Citation 'SourceSerifPro:2014' on page 2 undefined on input line 74
  Citation 'Topalidou:2015' on page 1 undefined on input line 4
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target 'article.bbl'.  Stop.
Rule 'biber article': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.bcf'
------------
Run number 1 of rule 'biber article'
------------
------------
Running 'biber  "article"'
------------
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.bbl'
      'article.out'
------------
Run number 2 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.run.xml'
------------
Run number 3 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.run.xml'
------------
Run number 4 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Build complete
```

If your build had any errors locally, please add it here. You can upload your build.log in a [Pastebin](https://pastebin.com/) link and put it here.

### CODE REVIEW

**Comment by Divyasha02** [comment]
Now i am facing only citation issue , pdf is generated without references and table references. following error is received:

enovo@lenovo-Lenovo-ideapad-330-15IKB:~/MLRC/2021$ ./build_all.sh
Building ahmed2022re/
/home/lenovo/MLRC/2021/ahmed2022re/journal
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl'
 No file article.bbl.
Latexmk: Missing input file: 'Capture.png' from line
  'LaTeX Warning: File `Capture.png' not found on input line 99.'
Latexmk: Missing input file: 'Capture.png' from line
  '! Unable to load picture or PDF file 'Capture.png'.'
Latexmk: Missing input file: 'graph_1.png' from line
  'LaTeX Warning: File `graph_1.png' not found on input line 311.'
Latexmk: Missing input file: 'graph_1.png' from line
  '! Unable to load picture or PDF file 'graph_1.png'.'
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation 'hartley2020dtoolai' on page 3 undefined on input line 34
  Citation 'he2016deep' on page 3 undefined on input line 44
  Citation 'he2016deep' on page 5 undefined on input line 67
  Citation 'he2016deep' on page 5 undefined on input line 69
  Citation 'heumos2021mlf' on page 3 undefined on input line 34
  Citation 'kornblith2019similarity' on page 7 undefined on input line 129
  Citation 'krizhevsky2009learning' on page 3 undefined on input line 44
  Citation 'loshchilov2016sgdr' on page 6 undefined on input line 122
  Citation 'paszke2019pytorch' on page 5 undefined on input line 67
  Citation 'renard2020variability' on page 3 undefined on input line 34
  Citation 'summers2021nondeterminism' on page 1 undefined on input line 3
  Citation 'summers2021nondeterminism' on page 1 undefined on input line 8
  Citation 'summers2021nondeterminism' on page 11 undefined on input line 342
  Citation 'summers2021nondeterminism' on page 9 undefined on input line 170
  Citation 'summers2021nondeterminism' on page 9 undefined on input line 177
  Citation 'szegedy2015going' on page 1 undefined on input line 3
  Citation 'wen2020batchensemble' on page 1 undefined on input line 3
  Citation 'zhuang2021randomness' on page 3 undefined on input line 34
  Citation 'zhuang2021randomness' on page 3 undefined on input line 47
  Reference `fig:pic0' on page 5 undefined on input line 69
  Reference `fig:pic1' on page 11 undefined on input line 276
  Reference `sec:AdditionalResults' on page 5 undefined on input line 58
  Reference `sec:Results' on page 12 undefined on input line 363
  Reference `table:table1' on page 5 undefined on input line 69
  Reference `table:table10' on page 11 undefined on input line 342
  Reference `table:table2' on page 5 undefined on input line 106
  Reference `table:table3' on page 7 undefined on input line 143
  Reference `table:table4' on page 9 undefined on input line 177
  Reference `table:table5' on page 9 undefined on input line 200
  Reference `table:table6' on page 10 undefined on input line 217
  Reference `table:table7' on page 10 undefined on input line 217
  Reference `table:table7' on page 10 undefined on input line 219
  Reference `table:table8' on page 11 undefined on input line 276
  Reference `table:table9' on page 11 undefined on input line 316
------------
Running 'make "Capture.png"'
------------
make[1]: *** No rule to make target 'Capture.png'.  Stop.
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target 'article.bbl'.  Stop.
------------
Running 'make "graph_1.png"'
------------
make[1]: *** No rule to make target 'graph_1.png'.  Stop.
Latexmk: Summary of warnings from last run of (pdf)latex:
  Latex failed to resolve 21 reference(s)
  Latex failed to resolve 23 citation(s)
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'article.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
make: *** [Makefile:30: article.pdf] Error 12
Build complete
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory
Building editorial/
/home/lenovo/MLRC/2021/editorial/journal
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl'
 No file article.bbl.
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation 'Roboto:2011' on page 2 undefined on input line 74
  Citation 'Rougier:2017' on page 1 undefined on input line 12
  Citation 'Rougier:2017' on page 1 undefined on input line 26
  Citation 'Rougier:2018' on page 1 undefined on input line 13
  Citation 'Science:2018' on page 1 undefined on input line 11
  Citation 'SourceCodePro:2012' on page 2 undefined on input line 74
  Citation 'SourceSerifPro:2014' on page 2 undefined on input line 74
  Citation 'Topalidou:2015' on page 1 undefined on input line 4
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target 'article.bbl'.  Stop.
Rule 'biber article': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.bcf'
------------
Run number 1 of rule 'biber article'
------------
------------
Running 'biber  "article"'
------------
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.bbl'
      'article.out'
------------
Run number 2 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.run.xml'
------------
Run number 3 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'article.aux'
      'article.run.xml'
------------
Run number 4 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Build complete

File: 2021/ahmed2022re/openreview/content.tex [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+		 & & \\
+		\hline
+	\end{tabular}
+	\caption{Architecture of ResNet-14 and ResNet-34 in modular format}
+	\label{table:table1}
+\end{table}
+
+\begin{figure}[!htb]
+	\centering
+	\includegraphics[width=0.25\textwidth]{Capture.png}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/content.tex:99]: This needs to be a relative import. Change to `../openreview/Capture.png` to make it work.

File: 2021/ahmed2022re/openreview/content.tex [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+		\hline
+	\end{tabular}
+	}
+	\caption{No. of models v/s Variability. N denotes total number of trained models.}
+	\label{table:table8}
+\end{table}
+
+\begin{figure}[!htb]
+	\centering
+	\includegraphics[width=0.5\textwidth]{graph_1.png}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/content.tex:311]: This needs to be a relative import. Change to `../openreview/graph_1.png` to make it work.

File: 2021/ahmed2022re/openreview/content.tex [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+\subsection{Model descriptions}
+We initially chose the ResNet-14 model because it was frequently used for image classification experiments in the original study along with the CIFAR-10 datasets. Due to the large number of trained models required for a single experiment, we did not have time to work with other models except ResNet-34, which we created ourselves to test the generalization of the claim over larger model architectures. Figure \ref{fig:pic0} shows the basic residual block of a ResNet architecture, consisting of two 3x3 convolutional layers followed by batch normalization before activation. On the same basis, Table \ref{table:table1} shows the modular architecture of ResNet-14 and ResNet-34 \cite{he2016deep} without the first 7x7 convolutional layer and the final fully-connected layer. The blocks are shown in parentheses along with the number of output channels, with the multiplier indicating the number of residual blocks in that module. 
+\begin{table}[!htb]
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/content.tex:70]: This table is also breaking compilation, due to use of `bmatrix`. Please debug this region or find an alternative way to display the same information.

File: 2021/ahmed2022re/openreview/content.tex [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+\begin{table}[!hb]
+\centering
+	\begin{tabular}{|c|c|c|c|c|}
+	\hline
+    Dataset & Classes & Samples& Dimensions& Split(train/val/test)\\
+	\hline
+		CIFAR-10 & 10 & 60k & $32 \times 32 \times 3$ & 50k/0/10K \\
+		\rowcolors{}{}{}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/content.tex:114]: remove this (rowcolors) - this is also breaking compilation

File: 2021/ahmed2022re/openreview/packages.tex [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+\usepackage{microtype}      % microtypography
+\usepackage{amsmath}
+\usepackage{caption}
+\captionsetup[table]{skip=5pt}
+\usepackage{float}
+\usepackage{graphicx}
+\usepackage{subcaption}
+\usepackage{hyperref}
+\usepackage{graphics}
+\usepackage{floatrow}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/packages.tex:32]: Remove the `floatrow` package.

**Comment by koustuvsinha** [comment]
Hi, I have added few comments to fix the issue. Additionally, these bib entries were not found in `journal/bibliography.bib` file:

```
Citation 'hartley2020dtoolai' on page 3 undefined on input line 34
  Citation 'he2016deep' on page 3 undefined on input line 44
  Citation 'he2016deep' on page 5 undefined on input line 67
  Citation 'he2016deep' on page 5 undefined on input line 69
  Citation 'heumos2021mlf' on page 3 undefined on input line 34
  Citation 'kornblith2019similarity' on page 7 undefined on input line 128
  Citation 'krizhevsky2009learning' on page 3 undefined on input line 44
  Citation 'loshchilov2016sgdr' on page 6 undefined on input line 121
  Citation 'paszke2019pytorch' on page 5 undefined on input line 67
  Citation 'renard2020variability' on page 3 undefined on input line 34
  Citation 'summers2021nondeterminism' on page 1 undefined on input line 3
  Citation 'summers2021nondeterminism' on page 1 undefined on input line 8
  Citation 'summers2021nondeterminism' on page 11 undefined on input line 341
  Citation 'summers2021nondeterminism' on page 8 undefined on input line 169
  Citation 'summers2021nondeterminism' on page 8 undefined on input line 176
  Citation 'szegedy2015going' on page 1 undefined on input line 3
  Citation 'wen2020batchensemble' on page 1 undefined on input line 3
  Citation 'zhuang2021randomness' on page 3 undefined on input line 34
  Citation 'zhuang2021randomness' on page 3 undefined on input line 47
 ```

Ensure that these references are in `journal/bibliography.bib` file.

File: 2021/ahmed2022re/openreview/content.tex (hunk 3 above)
Comment (koustuvsinha) [reply 2021/ahmed2022re/openreview/content.tex:70]: You can test it by removing this table (and fixing other errors) to get correct compilation.

**Comment by Divyasha02** [comment]
thank you! i will make corrections and will let you know.

**Comment by Divyasha02** [comment]
now the final PDF is ready, Kindly check and once again thanks for the help.

**Comment by wi25hoy** [comment]
@koustuvsinha Kindly review the updated version of Pull request. i am able to generate pdf now and latex file is working fine.

**Comment by wi25hoy** [comment]
@SharathRaparthy kindly review my pull request, i am able to generate correct version of pdf now , hence request you to update my status

**Comment by SharathRaparthy** [comment]
@Divyasha02 Thank you for the update! Everything looks good from my end.

**Comment by SharathRaparthy** [comment]
@wi25hoy Thank you for the update! Everything looks good from my end.

**Pull request merged**